step-function/cloudformation.yml

```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+
+  JobQueueArn:
+    Type: String
+  RtcGammaJobDefinition:
+    Type: String
+
+Outputs:
+
+  StepFunctionArn:
+    Value: !Ref StepFunction
+
+Resources:
+
+  StepFunction:
+    Type: AWS::StepFunctions::StateMachine
+    Properties:
+      RoleArn: !GetAtt StepFunctionRole.Arn
+      DefinitionString: !Sub |-
+        {
+          "StartAt": "BATCH_JOB",
+          "States": {
+           "BATCH_JOB": {
+             "Type": "Task",
+             "Resource": "arn:aws:states:::batch:submitJob.sync",
+             "Parameters": {
+               "JobDefinition": "${RtcGammaJobDefinition}",
+               "JobName.$": "$.parameters.granule",
```

Comment:
might jump straight to `$$.Execution.Name` for our batch job names now, so they're 1-1 with step function executions?  Or leave it out entirely?